Gems/AzQtComponentsForPython/Code/Platform/Mac/PAL_mac.cmake

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+set(PAL_TRAIT_AZQTCOMPONENTSFORPYTHON_SUPPORTED FALSE)
```

Comment:
Is it worth at least leaving a comment here as to why it is not supported and a hint as to what it would take to get it there?  Or a link to a ticket to add that support in case someone with a mac wants to have a run at it?